Emby.AutoOrganize/PluginEntryPoint.cs

```diff
@@ -17,13 +17,13 @@
 
 namespace Emby.AutoOrganize
 {
-    public class PluginEntryPoint : IServerEntryPoint
+    public sealed class PluginEntryPoint : IServerEntryPoint
```

Comment:
Why seal this class?

Reply:
To simplify the implementation for Dispose(). With an unsealed class you need to implement the full pattern with Dispose(bool). I can implement the full pattern if that is preferred, let me know.

Reply:
Ok, just wanted to hear the reasoning behind this. Since this shouldn’t ever be inherited it’s fine.